python_modules/libraries/dagster-shared/dagster_shared_tests/test_check.py

```diff
@@ -1689,13 +1688,11 @@ class MyTypedDict(TypedDict):
     (Optional[Set[str]], [{"a", "b"}], [{1, 2}]),
     (Optional[Dict[str, int]], [{"a": 1}], [{1: "a"}]),
     (Optional[Mapping[str, int]], [{"a": 1}], [{1: "a"}]),
-    (PublicAttr[Optional[Mapping[str, int]]], [{"a": 1}], [{1: "a"}]),  # type: ignore  # ignored for update, fix me!
-    (PublicAttr[Bar], [Bar()], [Foo()]),  # type: ignore  # ignored for update, fix me!
```

Comment:
i removed this because PublicAttr still lives in dagster and I wanted the tox suite to not include dagster, but can bring PublicAttr in if we think this test coverage is important